oxenstored/connections.ml

```diff
@@ -220,13 +235,52 @@ let fire_watches ?oldroot source root cons path recurse =
 
 let send_watchevents con = Connection.source_flush_watchevents con
 
-let fire_spec_watches root cons specpath =
+let fire_spec_watches root cons specpath domids =
   let source = find_domain cons 0 in
-  iter cons (fun con ->
-      List.iter
-        (Connection.fire_single_watch source (None, root) 0)
-        (Connection.get_watches con specpath)
-  )
+
+  let fire_spec_watch fire_generic domid =
+    (* Trigger @releaseDomain/domid watches as well *)
+    let specpaths =
```

Comment:
Do we have multiple predicates to recognise special paths? Would it make sense to lift them to an outer level?